src/database/UIDatabase.js

```diff
@@ -262,9 +262,14 @@ class UIDatabase {
     return this.database.write(...args);
   }
 
+  getPreference(key) {
+    const preference = this.database.get('Pref', key);
+    return preference?.value;
```

Comment:
Hmm, what happens to all the callers of getPreference if preference is not defined ?

Reply:
Food for though, I think it would be better to cast preference type somewhere here rather then in 'Pref' data type.

Reply:
Yeah, I agree, see 48c5aaab4a36171a98dc48edf55a688804713fab.